### PR TITLE
Michielkallenberg/inception time

### PR DESCRIPTION
**Pull request by michielkallenberg**
Issue #304

### CODE REVIEW

File: cybench/models/nn_models.py [hunk 1]
```diff
@@ -22,6 +22,21 @@
 )
 
 
+def separate_ts_static_inputs(batch: dict) -> tuple:
+    """Stack time series and static inputs separately.
+
+    Args:
+      batch (dict): batched inputs
```
Comment (krsnapaudel) [inline cybench/models/nn_models.py:29]: Maybe this could go to util/torch.py

File: cybench/models/nn_models.py [hunk 2]
```diff
@@ -606,26 +621,105 @@ def fit(
...
+            x = transform(x, self._min_date, self._max_date)
+
+        x_ts, x_static = separate_ts_static_inputs(x)
+        x_ts, _ = self._lstm(x_ts)
+        x = torch.cat([x_ts[:, -1, :], x_static], dim=1)
+        output = self._fc(x)
+        return output
+
+
+class ExampleInceptionTime(BaseNNModel):
```
Comment (krsnapaudel) [inline cybench/models/nn_models.py:635]: I don't like Example in the name. Both LSTM and InceptionTime could be renamed. Baseline is probably better than Example.

**Review by krsnapaudel — CHANGES_REQUESTED** [review]
Some small fixes.

**Review by krsnapaudel — APPROVED** [review]

File: cybench/models/nn_models.py (hunk 2 above)
Comment (aikepotze) [reply cybench/models/nn_models.py]: Let's change it to LSTMModel or LSTM

**Review by aikepotze — COMMENTED** [review]
Looks good to me! We might need to change the default hyperparameter tuning ranges, but implementation looks fine.

**Pull request merged**